web/themes/custom/server_theme/templates/server-theme-person-card.html.twig

```diff
@@ -0,0 +1,65 @@
+{#
+/**
+ * @file
+ * Person Card component.
+ *
+ * Variables:
+ * - image_url: URL of the person's image
+ * - name: Person's name
+ * - role: Person's role or title
+ * - description: Brief description or bio
+ */
+#}
+
+{#
+/**
+ * @file
+ * Person Card component.
+ *
+ * Variables:
+ * - image_url: URL of the person's image
+ * - name: Person's name
+ * - role: Person's role or title
+ * - is_admin: Boolean indicating if person is an admin
+ */
+#}
+
+<div class="bg-white rounded-lg shadow-md flex flex-col items-center text-center border border-gray-200 shadow-lg">
```

Comment:
This isn't aligned with the rest of the codebase - I'd expect to see composition and re-use of other existing/new elements.